docs/getting-started-on-snowplow-open-source/quick-start/index.md

```diff
@@ -615,4 +628,108 @@ For solutions to some common Terraform errors that you might encounter when runn
 
 :::
 
+## Configure the destination
+
+<Tabs groupId="warehouse" queryString>
+  <TabItem value="postgres" label="Postgres" default>
```

Comment:
Also not the main point of your PR but....

To me it looks out-of-place that Postgres is part of the quick start.  It made sense 2 years ago, because we had no other viable streaming loader.  But now we have multiple other loaders in the quick start, I don't think this should even be part of the product that we advertise.

Reply:
Discussed on Slack — keeping Postgres for the time being